rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -49,6 +49,11 @@ void fmt_class_string<CellSaveDataError>::format(std::string& out, u64 arg)
 	});
 }
 
+struct save_data_info
+{
+	std::atomic<bool> enable_overlay = false;
```

Comment:
Please use atomic_t

Reply:
Can't you reuse savedata_mutex struct? maybe rename it.

Reply:
I mean instead of save_data_info, not the boolean variable.

Reply:
You just bring confusion with unrelated structs, unrelated purpose. Only need to replace std::atomic with atomic_t, that's all.

Reply:
I think you misunderstood.

Reply:
Any atomic is enough to satisfy requirement (non-copyable), maybe you forgot or misunderstood it.

Reply:
What I mean is having less FXO-manadged structures for savedata because one exists already.